README.md

```diff
@@ -99,3 +102,60 @@ PKCS11 Identity using ATECC608B TrustAndGo
   "cert_id": "pkcs11:token=MCHP;object=device;type=cert"
 }
 ```
+
+### Configuring with Elixir
+
+The peridiod application can be set using config.exs in a Nerves based application. The following is an example of the keys that can be set:
+
+```elixir
+config :peridiod,
+  device_api_host: "device.cremini.peridio.com",
+  device_api_port: 443,
+  device_api_sni: "device.cremini.peridio.com",
+  device_api_verify: :verify_peer,
+  device_api_ca_certificate_path: nil,
+  key_pair_source: "env",
+  key_pair_config: %{"private_key" => "PERIDIO_PRIVATE_KEY", "certificate" => "PERIDIO_CERTIFICATE"},
+  fwup_public_keys: [],
+  fwup_devpath: "/dev/mmcblk0",
+  fwup_env: [],
+  fwup_extra_args: [],
+  remote_shell: false,
+  remote_iex: true,
+```
+
+## Running with Docker
+
+You can debug using docker by generating an SSL certificta eand private key pair that is trusted by Peridio Cloud and pass it into the container.
+
+Building the container:
+
+```bash
+docker build --tag peridio/peridiod .
```

Comment:
Is this intentionally `docker` over `podman` due to the build error you mentioned in a call earlier?